massa-ledger-worker/src/ledger_db.rs

```diff
@@ -371,7 +382,11 @@ impl LedgerDB {
 
```

Comment:
the length of the key should also be checked

Reply:
Done